src/root-config-dist.js

```diff
@@ -5,3 +5,35 @@ export * from "./root-config-lib";
 
```

Comment:
@joeldenning this workaround seems ok, but the travis build seems to be failing, could it be related to this?